org-roam-mode.el

```diff
@@ -169,32 +169,48 @@ which visits the thing at point."
         (magit-insert-heading)
         (run-hook-with-args 'org-roam-mode-section-functions org-roam-current-node)))))
 
-(defun org-roam-buffer ()
-  "Launch an Org-roam buffer for the current node at point."
-  (interactive)
-  (if-let ((node (org-roam-node-at-point))
-           (source-org-roam-directory org-roam-directory))
-      (progn
-        (let ((buffer (get-buffer-create
-                       (concat "org-roam: "
-                               (file-relative-name (buffer-file-name) org-roam-directory)))))
-          (with-current-buffer buffer
-            (org-roam-mode)
-            (setq-local org-roam-current-node node)
-            (setq-local org-roam-current-directory source-org-roam-directory)
-            (org-roam-buffer-render))
-          (switch-to-buffer-other-window buffer)))
-    (user-error "No node at point")))
+;;; Node dedicated buffer
+(defvar org-roam-dedicated-buffer-prefix "*org-roam: "
```

Comment:
It's a good rule to keep the functions and variables prefixed with the file name, in this case `org-roam-buffer`. Might be a bit strange sounding but I think `org-roam-buffer-dedicated-prefix` is more appropriate.

Reply:
Also, maybe use -format instead of prefix and use `format`? So people can choose whether anything should be added to the suffix (or at least close the * that is opened in the prefix)

Reply:
Wish it would be `org-roam-buffer.el`, but it's named as `org-roam-mode.el`. But I've got you 👍 
As for the prefix variable, I want to expose it to people for programmatic matching in Elisp, so putting the formatting specifiers would make it a bit difficult for people to use it for such purpose. I'll see what can be done about this.